guesser.rb

```diff
@@ -0,0 +1,41 @@
+# Create guesser.rb.  When this program starts, it asks you to think of a number from 1 to 100.  It then makes guesses.  You tell it whether the guess is too low, too high, or correct, until it guesses your number.  The program will need to keep track of the lowest and highest possible remaining values, and it should guess something between them.  The program should prompt you again if you say something besides "too low", "too high" or "correct".  The program should also handle the case where you are lying, and should call you out.  You are lying if the program correctly guesses the number, but you don't say so.  You are lying if you tell the program that its guess is too low, when it's really too high, or vice versa.  Of course, the program won't be able to tell right away that you are lying, as it will have to exhaust the possible answers.  Make the program efficient -- basically a binary search.  Once it guesses correctly, or figures out you are lying, it should prompt you to play again, exiting if you decline.
+
+
+loop do
+    # puts secret_number # Uncomment to get the output of the secret number
+
+    puts "Think of a number between 1 and 100"
+    guess = gets.chomp.to_i

```

Comment:
This is pretty close! The guess variable is being set on line 8 and line 14. Guess is being created through the high and low bounds so it doesn't need input from the user, in this case we can remove line 8.

Reply:
Fixed!